content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/quickstart-for-configuring-your-minio-storage-bucket-for-github-actions.md

```diff
@@ -0,0 +1,130 @@
+---
+
+Before you can enable and configure {% data variables.product.prodname_actions %} on {% data variables.product.product_location_enterprise %}, you need to prepare your third-party storage solution.
+
+MinIO offers object storage with support for the S3 API and {% data variables.product.prodname_actions %} on your enterprise.
+
+This quickstart shows you how to set up MinIO using Docker for use with {% data variables.product.prodname_actions %} but you have other options for managing MinIO besides Docker. For more information about MinIO, see the official [MinIO docs](https://docs.min.io/).
+
+## 1. Choose a MinIO mode for your needs
+
+| MinIO mode | Optimized for | Storage infrastructure required |
+|----|----|----|
+| Standalone MinIO (on a single host) | Fast setup |  N/A |
+| Clustered MinIO (also called Distributed MinIO)|  Data security | Storage servers running in a cluster |
+
+For more information about your options, see the official [MinIO docs](https://docs.min.io/).
+
+{% warning %}
+
+**Warning**: MinIO has announced removal of MinIO Gateways. Starting June 1st, 2022, support and bug fixes for the current MinIO NAS Gateway implementation will only be available for paid customers via their LTS support contract. If you want to continue using MinIO Gateways with {% data variables.product.prodname_actions %}, we recommend moving to MinIO LTS support. For more information, see [Scheduled removal of MinIO Gateway for GCS, Azure, HDFS](https://github.com/minio/minio/issues/14331) in the minio/minio repository.
+
+Other modes of MinIO remain available with standard support.
+
+{% endwarning %}
+
+## 2. Install, run, and sign in to MinIO
+
+1. Set up your preferred environment variables for MinIO.
+
+    These examples use `MINIO_DIR`:
+    ```shell
+    $ export MINIO_DIR=$(pwd)/minio
+    $ mkdir -p $MINIO_DIR
+    ```
+
+2. Install MinIO.
+
+    ```shell
+    $ docker pull minio/minio
+    ```
+    For more information, see the official "[MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide)."
+
+3. Sign in to MinIO using your MinIO access key and secret.
+
+    {% linux %}
+    ```shell
+    $ export MINIO_ACCESS_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    # this one is actually a secret, so careful
+    $ export MINIO_SECRET_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    ```
+    {% endlinux %}
+
+    {% mac %}
+    ```shell
+    $ export MINIO_ACCESS_KEY=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    # this one is actually a secret, so careful
+    $ export MINIO_SECRET_KEY=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    ```
+    {% endmac %}
+
+    You can access your MinIO keys using the environment variables:
+
+    ```shell
+    $ echo $MINIO_ACCESS_KEY
+    $ echo $MINIO_SECRET_KEY
+    ```
+
+4. Run MinIO in your chosen mode.
+
+   * Run MinIO using Docker on a single host:
```

Comment:
Are we continuing these prerequisites? - https://docs.github.com/en/enterprise-server@3.2/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-gateway-for-nas-storage#prerequisites

Particularly, `To avoid resource contention on the appliance, we recommend that MinIO be hosted separately from your GitHub Enterprise Server instance.`

Reply:
Apologies! I see that the prerequisites are mentioned in the other page.